xml/schema/Contact/DashboardContact.xml

```diff
@@ -56,7 +56,7 @@
   </index>
   <field>
     <name>column_no</name>
-    <type>boolean</type>
+    <type>int</type>
```

Comment:
I understand why they did this originally. `boolean` translates to `tinyint` in mysql (range 0-255), which I guess saves a few bytes in the database since `column_number` is always going to be a very small number. But now that  APIv4 is stricter about field types, this gets output as `true` and `false` by api.get... which is :hankey: 

Too bad our schema xml doesn't allow a `tinyint unsigned` field type, but this will do. Uses a few more bytes of disc space but whatever.

Strictly speaking this requires updating the schema sql and a database upgrade, except it doesn't matter since this change is just for the benefit of php not sql.